pkgs/development/libraries/libnetfilter_queue/default.nix

```diff
@@ -1,11 +1,11 @@
 { stdenv, fetchurl, pkgconfig, libmnl, libnfnetlink }:
 
 stdenv.mkDerivation rec {
-  name = "libnetfilter_queue-1.0.2";
+  name = "libnetfilter_queue-1.0.3";
 
   src = fetchurl {
     url = "ftp://ftp.netfilter.org/pub/libnetfilter_queue/${name}.tar.bz2";
```

Comment:
the https download link seems more appropriate `https://www.netfilter.org/projects/libnetfilter_queue/files/libnetfilter_queue-1.0.3.tar.bz2` to me.